plugins/nestjs-nats/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@amplication/plugin-nestjs-nats",
-    "version": "0.0.1-beta.0",
+    "version": "0.0.1-beta.1",
```

Comment:
Why still beta?

Reply:
Because for now, this plugin is still not fully-fledged plugin